sdk/src/main/scala/com/horizen/storage/SidechainHistoryStorage.scala

```diff
@@ -94,6 +94,11 @@ class SidechainHistoryStorage(storage: Storage, sidechainTransactionsCompanion:
     }
   }
 
+  //Block info shall be in history storage, otherwise something going totally wrong
+  def blockInfoByIdFromStorage(blockId: ModifierId): SidechainBlockInfo = {
```

Comment:
as was discussed, feel free to name of this and `blockInfoById`

Reply:
done

Reply:
done